protected/config/common.php

```diff
@@ -6,4 +6,10 @@
  * @see http://docs.humhub.org/dev-environment.html
  */
 return [
+    'components' => [
+        'urlManager' => [
+            'showScriptName' => false,
+            'enablePrettyUrl' => true,
```

Comment:
Just to add a note here, if this is set to true then you'll also have to edit the `.htaccess.dist` file and change the name as well to reflect it else you'll end up with a frontend server error

https://github.com/humhub/humhub/blob/master/.htaccess.dist

Personally I think adding this here is a good thing, only thing I personally think is that it should be commented out or at least have `enablePrettyUrl` set to `false` if the above isn't changed

Reply:
@ArchBlood Thanks for the hin. The common.php  change should not longer part of this PR.